openedx/adg/common/mailchimp_pipeline/client.py

```diff
@@ -1,6 +1,4 @@
-"""
-Mailchimp client
-"""
+"""Mailchimp client"""
```

Comment:
As per [edx style guide](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings) the preferable style is 

```
"""
Docstring here
"""
```

Reply:
Do we prefer edx guidelines over pep8?

Reply:
I think pep8 does not give error or warning over this

Reply:
Yes, Actually it is preferred to use single line in pep8 guidelines if docstring is a one liner, It is also inconsistent in open edX code :)

I am changing to the edX preferred style for this PR.